js_modules/dagster-ui/packages/ui-core/src/graphql/schema.graphql

```diff
@@ -930,11 +930,13 @@ type AssetCheckEvaluation {
 }
 
 type AssetCheckEvaluationTargetMaterializationData {
-  storageId: Int!
+  storageId: BigInt!
   runId: String!
   timestamp: Float!
 }
 
+scalar BigInt
```

Comment:
I feel like should be using something like https://the-guild.dev/graphql/scalars/docs/scalars/big-int (the automatically-generated typing seems too broad/wrong?), but most of the files say to not modify them directly, so I've just left the autogenerated types. Maybe somebody knowledgeable about the UI module and GraphQL can weigh in?